grafana-plugin/src/containers/RotationForm/RotationForm.tsx

```diff
@@ -154,7 +154,7 @@ const RotationForm: FC<RotationFormProps> = observer((props) => {
       rolling_users: userGroups,
       interval: repeatEveryValue,
       frequency: repeatEveryPeriod,
-      by_day: repeatEveryPeriod === 1 || repeatEveryPeriod === 0 ? selectedDays : null,
+      by_day: (repeatEveryPeriod === 0 && repeatEveryValue === 1) || repeatEveryPeriod === 1 ? selectedDays : null,
```

Comment:
FYI, if I understand this correctly, it limits the repeat input value to 1 but the backend allows to choose a different value as long as the number is less than or equal to the length of selected days (same below).

Reply:
> length of selected days

but if you select  days not in a row (e.g. Mo We and Su) and how it should depend on shift start and shift end? It's good topic to discuss, there are still some questions

Reply:
I've removed `repeatEveryValue === 1` checking